concrete/attributes/user_selector/controller.php

```diff
@@ -99,4 +103,25 @@ public function exportValue(\SimpleXMLElement $akn)
             $avn = $akn->addChild('value', $user->getUserID());
         }
     }
+
+    public function validateForm($p)
+    {
+        return $p['value'] != false;
```

Comment:
in php8 and above this is going to throw an error
The form validators can pass null to this validateFrom [See here](https://github.com/concrete5/concrete5/blob/501f92d5e4d92668bad0bb718f1acfdb99476586/concrete/src/Attribute/StandardValidator.php#L39)

We should be checking exactly what is passed and not just assuming its an array which has 'value' set

Reply:
Wow... so we have to update all attribute controllers. Lot of them have same code.